map/framework.cpp

```diff
@@ -394,7 +394,8 @@ Framework::Framework()
   {
     unique_ptr<FeatureType> feature(new FeatureType());
     Index::FeaturesLoaderGuard const guard(m_model.GetIndex(), fid.m_mwmId);
-    guard.GetOriginalFeatureByIndex(fid.m_index, *feature);
+    if (!guard.GetOriginalFeatureByIndex(fid.m_index, *feature))
```

Comment:
It's up to you, of course, but why not to modify the signature of the callback to:
```c++
bool m_getOriginalFeatureFn(FeatureID const & id, FeatureType & ft);
```
?